tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -595,6 +599,12 @@ def init(self):
         if self.state.failure_count > self.max_restarts:
             self.logger.critical("Max restarts exceeded")
             return RunnerManagerState.error()
+        elif self.state.failure_count > 0 and self.max_restart_backoff:
+            # Subtract one so we start with 2**0 (i.e., 1)
+            base_backoff = min(self.max_restart_backoff, 2**(self.state.failure_count-1))
+            used_backoff = base_backoff + random.uniform(0, 1)
+            self.logger.info(f"Waiting {used_backoff:.2f}s before retrying restarting browser")
```

Comment:
nit: `before retrying restarting browser` -> `before retrying to restart browser` maybe? Or just `before restarting browser` works too.